docs/geedocs/answer/3499938.html

```diff
@@ -62,18 +66,17 @@
 <li>SUSE Linux Enterprise Server 10</li>
```

Comment:
This is for 5.2.0. Whatever testing happened for it is already in the past. My understanding is that Google is not supporting that any longer, and we're only supporting the open source version. So I'd say we should change the text to read that these older versions were tested on these OS distributions, but it shouldn't read that any of them are supported.

We should say only the open source version is supported on the couple of OS distributions we intend to support.

Actually, I'm not sure we should even have documentation about what versions Google has tested on, or used to support. Unless, someone is going to look for that in the GEE open source documentation, we could, perhaps, just remove all of this information, and only keep what's relevant to the open source version.